README.md

```diff
@@ -64,21 +64,21 @@ $ npm install uppy --save
 
 We recommend installing from npm and then using a module bundler such as [Webpack](http://webpack.github.io/), [Browserify](http://browserify.org/) or [Rollup.js](http://rollupjs.org/).
 
-Add CSS [uppy.min.css](https://unpkg.com/uppy/dist/uppy.min.css), either to `<head>` of your HTML page or include in JS, if your bundler of choice supports it — transforms and plugins are available for Browserify and Webpack.
+Add CSS [uppy.min.css](https://transloadit.edgly.net/releases/uppy/v0.22.0/dist/uppy.min.css), either to `<head>` of your HTML page or include in JS, if your bundler of choice supports it — transforms and plugins are available for Browserify and Webpack.
 
-Alternatively, you can also use a pre-built bundle, for example from [unpkg CDN](https://unpkg.com/uppy/). In that case `Uppy` will attach itself to the global `window.Uppy` object.
+Alternatively, you can also use a pre-built bundle, for example from Transloadit's CDN: Edgly. In that case `Uppy` will attach itself to the global `window.Uppy` object.
 
 > ⚠️ The bundle currently consists of most Uppy plugins, so this method is not recommended for production, as your users will have to download all plugins, when you are likely using just a few.
 
 1\. Add a script to the bottom of `<body>`:
 
 ``` html
-<script src="https://unpkg.com/uppy"></script>
+<script src="https://transloadit.edgly.net/releases/uppy/v0.22.0/dist/uppy.min.js"></script>
 ```
```

Comment:
It was kinda nice that it could point to the latest version automatically, can we achieve that with edgly?

Reply:
I think what we could do is have a “latest” version or similar that contains the latest version. Would that work for you?

Reply:
Having said that.. with other projects we have actually regretted such magic URLs because people will actually use them, and you you can no longer make any bc jeopardizing changes without risking breaking the ecosystem for a ton of folks out there

Reply:
So do you think manually upgrading is actually a better way to go? Risking that we’ll have like 100 sites using Uppy 0.24 after two years, when recent version is 1.2 and there are some nasty bugs in 0.24 that we are ashamed of :-D I’m torn.